package.json

```diff
@@ -1,26 +1,28 @@
 {
-  "name": "starter-projects",
+  "name": "dat-cross-dam",
   "version": "0.0.1",
-  "description": "Concord Consortium starter projects",
+  "description": "DAT-Cross Dam simulation: A simulation of water flow and diversion using a dam on nearby infrastructure",
```

Comment:
Nice catch -- I'd left all the boilerplate along.

Reply:
There's more of it elsewhere - particularly in the Travis settings - which will come in a future PR :)